### ISSUE TITLE
2.	My Profile : Inconsistency in the Menu and the detail page. 

### ISSUE THREAD

**Issue by drasticdpk** [appirio-tech/connect-app#3779]
Left Navigation Menu>> My Profile >> Profile Information
Problem :  Menu title and the detail section header is not sync for Profile Information. 
Suggestion: Even the detail page should have the same Title as Profile Information. 
Benefit: It gives the clarity to the user as what information to be included.

### Screenshot/screencast
Attach or link a resource.

--
![inconsistency in profile](https://user-images.githubusercontent.com/25297859/76416063-766ece80-63c2-11ea-8468-0acfc91a4813.png)

#### Environment
- OS: Windows-10
- Browser (w/version): Chrome/ Version 80.0.3987.122 (Official Build) (64-bit))
- User role (client, copilot or manager): Copilot
- Account used: topcoderconnect+copilot@gmail.com

**Comment by vikasrohit** [comment]
@drasticdpk I don't think it is inconsistency, `Personal Information` is sub section of the `Profile Information`, I guess.
